api/src/pcapi/core/providers/api.py

```diff
@@ -6,6 +6,7 @@
 
 from pcapi.core import search
 from pcapi.core.logging import log_elapsed
+from pcapi.core.mails.transactional.pro.venue_provider_deleted import send_venue_provider_deleted_email
```

Comment:
(détail) Pour simplifier les imports de ces fonctions qui envoient des mails, on ajoute un import de la fonction dans `core/mails/transactional/__init__.py`, ce qui permet d'écrire : 

```suggestion
from pcapi.core.mails.transactional import send_venue_provider_deleted_email
```